db/migrate/20220225184023_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.integer :PostCounter
```

Comment:
- [ ] It will be nice to follow the ruby best practice for naming convention for methods and variables using snake_case. So I will suggest you change `:PostCounter` to `post_counter`